src/main/resources/generator/dependencies/pom.xml

```
@@ -54,7 +54,7 @@
     <jqwik.version>1.9.0</jqwik.version>
     <!-- Maven Plugins -->
     <os-maven-plugin.version>1.7.1</os-maven-plugin.version>
-    <protobuf-maven-plugin.version>0.6.1</protobuf-maven-plugin.version>
+    <protobuf-maven-plugin.version>2.3.0</protobuf-maven-plugin.version>
```

```suggestion
    <protobuf-maven-plugin.version>2.4.0</protobuf-maven-plugin.version>
```

As discussed, the regression you raised should now be fixed that was causing your build failures. Thanks for bearing with!

---

Thanks for the quick fix! :)